ogr/ogrsf_frmts/generic/ogrlayer.cpp

```diff
@@ -1530,6 +1530,72 @@ int OGRLayer::InstallFilter(OGRGeometry *poFilter)
 }
 //! @endcond
 
+/************************************************************************/
+/*                   DoesGeometryIntersectEnvelope()                    */
+/************************************************************************/
+
+static bool DoesGeometryIntersectEnvelope(const OGRGeometry *poGeometry,
```

Comment:
I think it would be more clear to call this something like `DoesGeometryHavePointInEnvelope` because it is not fully testing intersection.

Reply:
good point. done